packages/plugin-nested-docs/README.md

```diff
@@ -135,8 +135,8 @@ You can also extend the built-in `parent` and `breadcrumbs` fields per collectio
 
 ```js
 import { CollectionConfig } from "payload/types";
-import createParentField from "@payloadcms/plugin-nested-docs/fields/parent";
-import createBreadcrumbsField from "@payloadcms/plugin-nested-docs/fields/breadcrumbs";
+import createParentField from "@payloadcms/plugin-nested-docs/fields";
+import createBreadcrumbsField from "@payloadcms/plugin-nested-docs/fields";
```

Comment:
This is still not correct, there cannot be two default exports from the same file.

Reply:
^^ I've got these imports sorted out in #4462